bin/pygrb/pycbc_pygrb_pp_workflow

```diff
@@ -43,6 +43,48 @@ __date__ = pycbc.version.date
 __program__ = "pycbc_pygrb_pp_workflow"
 
 
+# Function used to retrive a File instance from a FileList based on an
+# injection set label
+def fish_label_in_filelist(label, filelist):
```

Comment:
This function doesn't belong here. It should be a method in the FileList class (in core.py). BUT first, does the functionality desired already exist? 

Are you basically asking here if `label` is a `tag` in the file? (Note that using the file `name` is dangerous and should not be done, checking the attributes is better). If so, this exists:

https://github.com/gwastro/pycbc/blob/master/pycbc/workflow/core.py#L1549

if not, can you elaborate on what label is (or what the file attributes are, to figure out how it decides this).